src/lib/components/market/item_row_wrapper.ts

```diff
@@ -137,6 +137,10 @@ export class ItemRowWrapper extends FloatElement {
         }
 
         if (this.itemInfo) {
+            if (this.asset && !isMarketSkin(this.asset)) {
```

Comment:
We can move this `if` to outside this block (after the inspect link check above)

Reply:
my thought process was we probably want to display the error/loading first right?

Reply:
otherwise i'd be happy to move it

Reply:
Nah, I think we should treat it the same as an item with no inspect link.

Reply:
oh right i forgot, was tunnel visioned into causing no regressions